backend/internal/storage/postgres/schema/review.go

```diff
@@ -0,0 +1,66 @@
+package user
+
+import (
+	"context"
+	"platnm/internal/models"
+
+	"time"
+
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+type ReviewRepository struct {
+	db *pgxpool.Pool
+}
+
+func (r *ReviewRepository) GetReviewsByUserID(ctx context.Context, id string) ([]*models.Review, error) {
+
+	rows, err := r.db.Query(ctx, "SELECT * FROM review WHERE user_id = $1", id)
+	if err != nil {
+		print(err.Error(), "from transactions err ")
+		return []*models.Review{}, err
+	}
+	defer rows.Close()
+
+	var reviews []*models.Review
+	for rows.Next() {
+
+		var review models.Review
+
+		var mediaType, comment, userID, mediaID, rating *string
```

Comment:
i dont think u need these temp vars. u can bind directly to fields of review struct that u define on line 28

Reply:
see [this](https://pkg.go.dev/github.com/jackc/pgx/v5#example-RowToStructByName) from the docs